frontend/src/exploration/previews/Previews.tsx

```diff
@@ -1,20 +1,48 @@
 import React from 'react';
-import { Box, makeStyles } from '@material-ui/core';
-import { blue } from '@material-ui/core/colors';
+import { Box, List, makeStyles, Paper } from '@material-ui/core';
 import { createStyles } from '@material-ui/core/styles';
+import LayoutCard from './LayoutCard';
+import useService from '../../dependency-injection/useService';
+import ExplorationStore from '../../stores/exploration/ExplorationStore';
+import useObservable from '../../utils/useObservable';
+import layoutsData from './layoutsData';
+import { ExplorationWeight } from '../../stores/exploration';
 
 const useStyle = makeStyles(() =>
   createStyles({
     container: {
-      backgroundColor: blue[500], // TODO change/remove
-      height: '75vh', // TODO change/remove
+      height: '100%',
+      width: '100%',
     },
   })
 );
 
+/**
+ * List of {@link LayoutCard}s with ordering based on a {@link ExplorationWeight} from {@link ExplorationStore}.
+ */
 function Previews(): JSX.Element {
   const classes = useStyle();
-  return <Box className={`${classes.container} Previews`}>TODO</Box>;
+
+  const explorationStore = useService(ExplorationStore);
+
+  const weights = useObservable(
+    explorationStore.getScoreState(),
+    explorationStore.getScoreValue()
+  );
+
+  const layoutsPreviewData = (
+    Object.keys(weights) as (keyof ExplorationWeight)[]
+  )
+    .sort((a, b) => weights[a] - weights[b])
+    .map((layout) => layoutsData[layout]);
+
+  return (
+    <Box className={`${classes.container} Previews`}>
```

Comment:
Header from Issue missing.
```suggestion
    <Box className={`${classes.container} Previews`}>
          <h1>Recommended Visualisations</h1> // title approved by POs
```